pymc3/smc/sample_smc.py

```diff
@@ -157,6 +147,29 @@ def sample_smc(
         %282007%29133:7%28816%29>`__
     """
 
+    if kernel is not None:
+        warnings.warn(
+            "The argument kernel in sample_smc has been deprecated. It is no "
+            "longer needed to specify it.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+    if save_sim_data is not None:
```

Comment:
We still want this, as doing `pm.sample_posterior_predictive` could be potentially too expensive.

Reply:
We no longer have access to the simulated data in the `logp` graph